BPOR/BPOR.Rms/Controllers/EmailController.cs

```diff
@@ -9,29 +9,28 @@ public Task<IActionResult> SetupCampaign()
     {
         return Task.FromResult<IActionResult>(View());
     }
-    
+
     [HttpPost]
-    public async Task<IActionResult> SetupCampaign(SetupCampaignViewModel model)
+    public IActionResult SetupCampaign(SetupCampaignViewModel model)
```

Comment:
Although this isn't async atm it will become

Reply:
This is a minor one. I wanted to resolve the current build warning. This can be made async again when needed.